packages/cli/test/commands/account.test.ts

```diff
@@ -595,6 +593,44 @@ describe('account command group', () => {
     })
   })
 
+  describe('createConfigFromLoginParameters() helper', () => {
+    describe('when login parameters are marked required', () => {
+      const credentialsType = {
+        elemID: new ElemID('MockAdaptorName'),
+        fields: {
+          username: {
+            annotations: {
+            }
+          },
+          token: {
+            annotations: {
+              _required: true
+            }
+          }
+        }
+      }
```

Comment:
the input for the function should be an `ObjectType`
```suggestion
      const credentialsType = new ObjectType({
        elemID: new ElemID('MockAdaptorName'),
        fields: {
          username: {
            annotations: {
            }
          },
          token: {
            annotations: {
              _required: true
            }
          }
        }
      })
```

Reply:
Done.